### DISCUSSION TITLE
Run `go mod tidy` after rebase

### DISCUSSION THREAD

**Discussion by enocom** [renovatebot/renovate#9272] [Request Help]
We often have multiple PRs from renovate bot at once. When we start merging them, renovate will rebase onto the `main` branch, but it seems it won't re-run `go mod tidy`, even though we've [configured it to do so](https://github.com/GoogleCloudPlatform/cloudsql-proxy/blob/main/.github/renovate.json#L12). As a result, we then have to manually run the command and amend the commit before we can merge.

Would it be possible to run `go mod tidy` after rebasing?

Here's an example PR where we hit this issue: https://github.com/GoogleCloudPlatform/cloudsql-proxy/pull/699.

/cc @shubha-rajan

**Comment by HonkingGoose** [comment]
I think issue https://github.com/renovatebot/renovate/issues/6213 is related to your problem?

**Reply by enocom** [reply]
Yep -- that's it. Thank you!

**Reply by rarkins** [reply]
FWIW this issue is really perplexing. Somehow.. either "go mod tidy" is not doing its job *or* it's being silently skipped.
